src/components/_modal/pages/base/localizationsBase/LocalizationsBase.js

```diff
@@ -143,6 +139,22 @@ export function Information({ children, className }) {
   );
 }
 
+/**
+ * Compound subcomponent Information for {@link LocalizationsBase} which can contain additional information on branches/agencies
+ * @param children
+ * @param {string} className
+ * @returns {JSX.Element}
```

Comment:
fremover bliver det React.ReactElement | null i stedet for JSX.Element, som er deprecated og children vil så være {Object}, tænker jeg?

Reply:
Min er gladest hvis children er "React.ReactNode | null"

Reply:
Beklager, nu har jeg lige fundet ud af vi kan gå med React.JSX.Element, hvis vi installerer types/react. JEg tænker, vi går bare med det.

Reply:
medmindre den broker sig hos dig over React.JSX.Element?

Reply:
Gør den vel indtil vi har installeret types/react :-D Men lad os gå med det. Jeg må rette det når du har merget din ind i main